docs/SOPs/dataset_wrangling_SOP.md

```diff
@@ -176,14 +176,20 @@ A request for an account should be files as [a new ticket](https://github.com/eb
 
 ### Data upload Procedure
 These commands can be run by team members with "developer" role.
-- create an AWS user for them
+- create an AWS user for the contributor
   
 ```shell
-# to add user alice as a contributor
-aws iam create-user --user-name alice --tags Key=project,Value=hca Key=owner,Value=tburdett Key=service,Value=ait
-aws iam add-user-to-group --group hca-contributor --user-name alice
+# to add user as a contributor, as convention we user the email prefix up to the '@'
+echo "Enter contributor account name:"
+read ACCOUNT
+aws iam create-user --user-name %ACCOUNT% --tags Key=project,Value=hca Key=owner,Value=tburdett Key=service,Value=ait
+aws iam add-user-to-group --group hca-contributor --user-name %ACCOUNT%
 # generate secrets 
-aws iam create-access-key --user-name alice 
+aws iam create-access-key --user-name %ACCOUNT% > %ACCOUNT%.txt
+# create a password protected zip file
+zip -e %ACCOUNT%.zip %ACCOUNT%.txt
```

Comment:
Why do we need to create a zip file? we usually email the contributor with the keys directly, without sending the file 

In case this is because of a security issue, we need to either

* Modify the file so that it only contains the keys that are useful for the contributors, no extra info
* Modify the upload [upload instructions](https://github.com/ebi-ait/hca-documentation/wiki/How-to-upload-data-to-an-upload-area-using-hca-util) so that the fields they need to extract from the file are called exactly as depicted in the documentation


Also, now that I look at the upload instructions, should we have some appendix on how to retrieve existing contributor info? e.g. a contributor wants their credentials because they lost them

Reply:
The zip is password protected for security indeed.
I will modify the script ass you suggested.

Reply:
Lost access keys cannot be restored. In such a case, the contributor would need new keys generated.